artifact-management/commands/gem-test.yml

```diff
@@ -1,33 +1,35 @@
-description: Perform gem test with default only or many Ruby versions
+description: Perform gem test with system Ruby, or with one or more Ruby
+  versions (using RVM) if ruby_versions is set. ruby_versions should be a
+  comma-separated string with no spaces, such as "2.2.10,2.3.8,2.5.8"
 parameters:
-  packagecloud_read_token:
-    description: The token to be used when fetching packages from Packagecloud.
-    type: env_var_name
-    default: PACKAGECLOUD_READ_TOKEN
-  gem_repo_key:
-    description: The :rubygems_api_key used to access the gem repository host
-    type: env_var_name
-    default: AVVO_GEM_REPO_TOKEN
   bundler_1_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
+    description: The precise Bundler 1.x version
     type: string
     default: 1.17.3
   bundler_2_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
+    description: The precise Bundler 2.x version
```

Comment:
I meant "The precise Bundler 2.x version _that should be used..._". The rest of the sentence was important! I was just providing an edit to the (ambiguous phrase) "The Bundler version" which should instead be "The precise Bundler 1.x version" or "The precise Bundler 2.x version"